examples/advanced/xgboost_secure/train_standalone/train_federated.py

```diff
@@ -23,9 +24,34 @@
 import xgboost.federated
 
 PRINT_SAMPLE = False
-DATASET_ROOT = "/tmp/nvflare/xgb_dataset/vertical_xgb_data"
-TEST_DATA_PATH = "/tmp/nvflare/xgb_dataset/test.csv"
-OUTPUT_ROOT = "/tmp/nvflare/xgb_exp"
+
+
+def train_federated_args_parser():
+    parser = argparse.ArgumentParser(description="Train federated XGBoost model")
+    parser.add_argument("--world_size", type=int, default=3, help="Total number of clients")
+    parser.add_argument("--gpu", type=int, default=0, help="Whether to use gpu for training, 0 for cpu, 1 for gpu")
+    parser.add_argument(
+        "--vert", type=int, default=0, help="Horizontal or vertical training, 0 for horizontal, 1 for vertical"
+    )
+    parser.add_argument(
+        "--enc", type=int, default=0, help="Whether to use encryption plugin, 0 for non-encrypted, 1 for encrypted"
+    )
+    parser.add_argument(
+        "--data_train_root",
+        type=str,
+        default="/tmp/nvflare/xgb_dataset/base_xgb_data",
+        help="Path to training data folder",
+    )
+    parser.add_argument(
+        "--data_test_file", type=str, default="/tmp/nvflare/xgb_dataset/test.csv", help="Path to testing data file"
+    )
+    parser.add_argument(
+        "--out_path",
+        type=str,
+        default="/tmp/nvflare/xgb_exp/base",
+        help="Output path for the data split file",
```

Comment:
suggest  "/tmp/nvflare/xgboost_secure/train_standalone" , this is minor, we just need to make sure all the XGBoost related examples not using conflicting default path

Reply:
updated the defults (the actual out_dir remain unchanged in the run_training_standalone script)